src/kvstore/RocksEngine.h

```diff
@@ -206,11 +206,13 @@ class RocksEngine : public KVEngine {
   /*********************
    * Non-data operation
    ********************/
-  void addPart(PartitionID partId) override;
+  nebula::cpp2::ErrorCode persistPart(PartitionID partId, const std::string& info) override;
+
+  void addPart(PartitionID partId, const HostAddr& learner = HostAddr::nullAddr()) override;
```

Comment:
It is best to add a comment to describe the difference between the learner parameters of ordinary peer and learner